crates/locutus-core/src/lib.rs

```diff
@@ -8,6 +10,7 @@ mod operations;
 mod ring;
 pub mod util;
 
+#[allow(dead_code)]
```

Comment:
if this is not needed anymore, let's remove it

Reply:
Done